RecordRTC.js

```diff
@@ -4808,6 +4811,7 @@ function GifRecorder(mediaStream, config) {
         var video = document.createElement('video');
         video.muted = true;
         video.autoplay = true;
+        video.playsInline = true;
```

Comment:
this is unrelated to this PR, but was created during the `grunt` process. This line is likely from #572